src/components/MapRoadSingleLane.js

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import {Image, View} from "react-native"
+import Lane from "./Lane"
+import Headquarters from "./Headquarters"
+import singleLineMap from "../assets/map_single_line.png"
+import styles from "../styles"
+import Touchable from "./Touchable"
+import Map from "./Map"
+import engine from "../gameLogic/Engine"
+
+export default class MapRoadSingleLane extends Map {
+
+  horizontalLane(progress) {
+    const margins = 35
+    const centerY = this.height / 2
+    const verticalOffset = 30
+    return {
+      x: progress * (this.width - margins * 2) + margins,
+      y: centerY + verticalOffset
+    }
+  }
+
+  renderMapContent() {
```

Comment:
This is related to the whole map / sizeaware thing that is a bit wonky and goes against the grain of react, but maybe we can talk about it through VOIP.